qemu/tests/cfg/blockdev_mirror_qemuio_ready_job.cfg

```diff
@@ -26,3 +26,5 @@
     storage_type_default = directory
     qemuio_cmd = '%s -f %s %s'
     error_msg = Failed to get "write" lock
+    ceph:
```

Comment:
filesystem only for this case, please make a double check if we want to extend the storage backends

Reply:
Have removed it, thanks.

Reply:
Have updated it, thanks zhenchao.